src/redux/Categories/Categories.js

```diff
@@ -0,0 +1,16 @@
+export const addCategory = () => ({
+  type: 'CHECK_STATUS',
+});
+
+/* eslint-disable default-param-last */
+
+const categoriesReducer = (state = 'Check Status', action) => {
```

Comment:
- [ ] From the requirements of this project, the initial state should be an empty array not a string as in your case.